fe/fe-core/src/test/java/com/starrocks/journal/bdbje/BDBEnvironmentTest.java

```diff
@@ -482,4 +484,49 @@ protected void testAddBadFollowerBase(boolean failover) throws Exception {
             LOG.warn("==============> getDatabasesNames() {}", masterEnvironment.getDatabaseNames());
         }
     }
+
+    @Test
+    public void testGetDatabase() throws Exception {
+        String selfNodeHostPort = findUnbindHostPort();
+        BDBEnvironment environment = new BDBEnvironment(
+                createTmpDir(),
+                "standalone",
+                selfNodeHostPort,
+                selfNodeHostPort,
+                true);
+        environment.setup();
+
+        new MockUp<ReplicatedEnvironment>() {
```

Comment:
Better open real database here to verify.

Reply:
it's more suitable here to use mock.